drivers/kscan/kscan_handlers.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/kscan.h>
+#include <syscall_handler.h>
+
+Z_SYSCALL_HANDLER(kscan_config, dev, callback_isr)
+{
+	Z_OOPS(Z_SYSCALL_DRIVER_KSCAN(dev, config));
+	Z_OOPS(Z_SYSCALL_VERIFY_MSG(callback_isr == 0,
+				    "callback cannot be set from user mode"));
+	return z_impl_kscan_config((struct device *)dev, callback_isr);
+}
+
```

Comment:
I think you need to include the auto-generated marshalling functions...see other system calls for examples

Reply:
Ok, I'm adding the syscalls_xxx_mrsh.c for each handler and rework the sample app for  kscan

Reply:
done